src/it/GetAndUnpackWithIncludes/pom.xml

```diff
@@ -0,0 +1,38 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <!-- Build description -->
+    <groupId>com.googlecode.maven-download-plugin.it</groupId>
+    <artifactId>testGetAndUnpackWithIncludes</artifactId>
+    <packaging>pom</packaging>
+    <version>${testing.versionUnderTest}</version>
+    <name>Test</name>
+
+    <!-- Build plugins and extensions -->
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>@project.groupId@</groupId>
+                <artifactId>@project.artifactId@</artifactId>
+                <version>@project.version@</version>
+                <executions>
+                    <execution>
+                        <phase>generate-resources</phase>
+                        <goals>
+                            <goal>wget</goal>
+                        </goals>
+                        <configuration>
+                            <!-- downloading a locally hosted jar archive, containing a "Hello, world!" text -->
+                            <url>@mrm.repository.url@/localhost/hello/1.0/hello-1.0.jar</url>
+                            <unpack>true</unpack>
+                            <includes>
+                                <include>hello.*</include>
```

Comment:
@smaarn would be super nice to have 3 files in jar/zip
- to_be_included_1
- to_be_included_2
- to_be_not_included

Reply:
Indeed. Reworked the reference resource to actually also include directories.